<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualizerTitleBar</class>
 <widget class="QWidget" name="VisualizerTitleBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>26</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>26</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>26</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VisualizerTitleBar</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#VisualizerTitleBar {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255))
 }
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Icons.qrc">:/resources/icons/wizualizacja2.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboType">
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAlphabetically</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameSources">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="layoutSources">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameCustom">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="layoutCustom">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="buttonSplitV">
     <property name="text">
      <string>split v</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonSplitH">
     <property name="text">
      <string>split h</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonFloat">
     <property name="text">
      <string>o</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="buttonClose">
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/resources/icons/oddokowywanie.png</normaloff>
     <normalon>:/resources/icons/dokowanie.png</normalon>:/resources/icons/oddokowywanie.png</iconset>
   </property>
   <property name="text">
    <string>Float</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/resources/icons/x.png</normaloff>:/resources/icons/x.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close</string>
   </property>
  </action>
  <action name="actionSplitVertically">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/resources/icons/vertical.png</normaloff>:/resources/icons/vertical.png</iconset>
   </property>
   <property name="text">
    <string>Split vertically</string>
   </property>
  </action>
  <action name="actionSplitHorizontally">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/resources/icons/horizontal.png</normaloff>:/resources/icons/horizontal.png</iconset>
   </property>
   <property name="text">
    <string>Split horizontally</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>comboType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>VisualizerTitleBar</receiver>
   <slot>setCurrentVisualizer(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setCurrentSlot(int)</slot>
  <slot>setCurrentVisualizer(int)</slot>
 </slots>
</ui>
